keyboards/momokai/aurora/info.json

```diff
@@ -0,0 +1,74 @@
+{
+    "manufacturer": "momokai",
+    "keyboard_name": "momokai/aurora",
+    "maintainer": "peepeetee",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "rgblight": false
+    },
+    "bootmagic": {
+        "matrix": [1, 0]
+    },
+    "matrix_pins": {
+        "direct": [
+            ["NO_PIN", "B0", "NO_PIN" ],
+            ["B4", "D2", "D1" ],
+            ["D7", "D4", "D6" ]
+        ]
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0009",
+        "vid": "0x69F9"
+    },
+    "encoder": {
+        "enabled": true,
+        "rotary": [
+            {
+                "pin_a": "B2",
+                "pin_b": "B1"
+            }
+        ]
+    },
+    "ws2812": {
+        "pin": "C7"
+    },
+    "rgb_matrix": {
+        "driver": "WS2812",
```

Comment:
```suggestion
        "driver": "ws2812",
```

Reply:
I had this on local but did not push, I have now pushed